### PR TITLE
Hello world linters

### PR DESCRIPTION
**Pull request by KDlamini**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+test.md
```
Comment (dasileker) [inline .gitignore:2]: - [ ] Please add the gitignore file also [see here](https://user-images.githubusercontent.com/61562626/128883503-07f8d8f7-7227-4f50-9407-0b780db78904.jpeg)

**Review by dasileker — CHANGES_REQUESTED** [review]
### **Hi @KDlamini** 

### **To highlight**

- The commit are good 
- the readme file is nite.

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: .gitignore (hunk 1 above)
Comment (KDlamini) [reply .gitignore:2]: Is this the only thing I have to fix then I will be good to proceed?
I have only one review left?

**Review by GzimAsani — APPROVED** [review]
## Hello @KDlamini 

### Your project is APPROVED :partying_face: :green_circle:. Good luck with your next projects :clap: :+1: 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- [ ] Your pull request should have a descriptive description, it is highly recommended that you add one.


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**